persist/sqlite/consensus.go

```diff
@@ -399,3 +399,7 @@ func setLastCommittedIndex(tx *txn, index types.ChainIndex) error {
 func insertAddressStatement(tx *txn) (*stmt, error) {
 	return tx.Prepare(`INSERT INTO sia_addresses (sia_address, siacoin_balance, immature_siacoin_balance, siafund_balance) VALUES ($1, $2, $2, $3) ON CONFLICT (sia_address) DO UPDATE SET sia_address=EXCLUDED.sia_address RETURNING id`)
 }
+
+func insertIndexStmt(tx *txn) (*stmt, error) {
+	return tx.Prepare(`INSERT INTO chain_indices (height, block_id) VALUES ($1, $2) ON CONFLICT (block_id) DO UPDATE SET height=EXCLUDED.height RETURNING id`)
```

Comment:
Similar question. Why would we ever need to update the height on a conflict? I'd expect a "DO NOTHING" here.

Reply:
DO NOTHING doesn’t execute the returning clause.